pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file:
+
+`composer init`
+
+- Create new project from an existing package:
+
+`composer create-project {{project_name}} {{target_path}} {{version}}`
```

Comment:
Ideally we'd have the simple version of this command without the version, and the more complete one that specifies it, noting the difference in the corresponding descriptions. But since this page is already hitting the size limit in our guidelines, I guess we could just tweak the description to say "...from a given version of an existing package". Does that sound reasonable?